src/api.js

```diff
@@ -39,19 +36,13 @@ const api = new ParseServer({
   push: {
     ios: {
       token: {
-        key: IOS_APN_KEY,
+        key: `/${IOS_APN_KEY}`,
```

Comment:
Not sure if this is correct but this needs to be the file path. 

`push: {
  ios: {
    token: {
      key: '/file/path/to/AuthKey_XXXXXXXXXX.p8',
      keyId: "XXXXXXXXXX",
      teamId: "YYYYYYYYYY" // The Team ID for your developer account
    },
    topic: 'com.domain.appname', // The bundle identifier associated with your app
    production: false
  }
}`

Reply:
IOS_APN_KEY should be the file path in heroku envs. I will double check

Reply:
I think we should leave it as it was